maproulette/src/main/java/com/pixeltron/mapquest/open/geocoding/RequestMaker.java

```diff
@@ -0,0 +1,35 @@
+package com.pixeltron.mapquest.open.geocoding;
+
+public class RequestMaker {
```

Comment:
The name of this class is potentially a bit misleading - the only interfaces it supports are really just _building_ requests, and not necessarily making them. A more descriptive update would be to make this `RequestBuilder`, or alternately update the `makeRequest` methods to return a `Future<HTTPResponse>` and call the fetch service within this class.